ui_framework/src/components/icon/icon.js

```diff
@@ -87,3 +88,7 @@ KuiIcon.propTypes = {
   size: PropTypes.oneOf(SIZES),
   title: PropTypes.string,
 };
+
+KuiIcon.defaultProps = {
+  className: 'kuiIcon--basic',
```

Comment:
Yeah, I was thinking that `kuiButton` by itself wouldn't be usable without a modifier. Bootstrap is kind of the same way, in that the default button just looks like a weird link.